controllers/consoleplugin_resource_reconciler.go

```diff
@@ -364,6 +364,13 @@ func (r *ConsolePluginResourceReconciler) setDesiredConsolePluginDeployment(
 			Protocol:      corev1.ProtocolTCP,
 		},
 	}
+
+	varTrue := true
+	consolePluginContainer.SecurityContext = &corev1.SecurityContext{
+		AllowPrivilegeEscalation: &varTrue,
+		ReadOnlyRootFilesystem:   &varTrue,
```

Comment:
I tried to install the current on OCP 4.10 (which has support for dynamic console plugins) and the console plugin container gets in a crashloop with the following error:

```shell
/opt/app-root/etc/generate_container_user: line 6: /opt/app-root/etc/passwd: Read-only file system
/usr/share/container-scripts/nginx/common.sh: line 16: cannot create temp file for here-document: Read-only file system
```
I think we shouldn't use a readonly filesystem for this component. Or do you think that there is anything better security-wise we could do?

Reply:
I've relaxed this, but I don't think it should create a user.
Let me see if we can change the console plugin container as a separate thread.

Reply:
By removing the readonly filesystem, it works as expected.